sound/soc/sof/intel/Kconfig

```diff
@@ -26,7 +26,6 @@ config SND_SOC_SOF_INTEL_PCI
 	select SND_SOC_SOF_COFFEELAKE  if SND_SOC_SOF_COFFEELAKE_SUPPORT
 	select SND_SOC_SOF_ICELAKE     if SND_SOC_SOF_ICELAKE_SUPPORT
 	select SND_SOC_SOF_COMETLAKE   if SND_SOC_SOF_COMETLAKE_SUPPORT
-	select SND_SOC_SOF_COMETLAKE_S if SND_SOC_SOF_COMETLAKE_S_SUPPORT
```

Comment:
ok, patch splitting looks a bit off. You could just merge these (so far) 2 patches or, if you really want to split them, maybe first add a new generic CML Kconfig option, then convert all uses of old ones to new ones in the code, finally remove old Kconfig options. But I think just 1 patch would be ok and easy to understand for all of these.

Reply:
@plbossart Yup, maybe make a series that reverts the non-upstream CML-S patch, then 2nd patch to unify the ifdefs for CML and final patch that simply adds the PCI ID for CML-S. Then we could just send patches 2 and 3 to upstream.

Reply:
ok, will add a revert for clarity